Dockerfile

```diff
@@ -136,7 +139,12 @@ RUN npm install \
 ###
 # pshtt
 
-RUN pip3 install pshtt==0.2.1
+RUN apt-get install -qq --yes locales
+RUN locale-gen en_US.UTF-8
+ENV LANG en_US.UTF-8
+ENV LANGUAGE en_US:en
+ENV LC_ALL en_US.UTF-8
+RUN pip3 install pshtt
```

Comment:
Locked to latest `pshtt`. This also required me to paste in please-just-make-it-work attempts at fixing a Unicode error that was happening because of the PyPi `publicsuffixlist` module's LICENSE file having Unicode characters in the author's name.